action.yml

```diff
@@ -13,6 +13,10 @@ inputs:
     description: number of times to attempt fetching the commit status
     required: false
     default: 10
+  issues_over_a11y_threshold:
```

Comment:
Why is this an input? I don't think this is correct.

Reply:
Because we have the UI side beside a feature flag, this was added as an input, a better name would be `ENABLE_A11Y_THRESHOLD` - can see 🧵  [here](https://deque.slack.com/archives/C04JQGWR351/p1688085118112639)